build.gradle

```diff
@@ -99,7 +99,7 @@ dependencies {
     testUtil sourceSets.test.output
 
     signature 'org.codehaus.mojo.signature:java18:1.0@signature'
-    signature 'net.sf.androidscents.signature:android-api-level-24:7.0_r2@signature'
+    signature 'net.sf.androidscents.signature:android-api-level-26:8.0.0_r2@signature'
```

Comment:
@TimvdLippe not really necessary (could be workarounded) but brings in these APIs [1] (seems like not part of v24) please let me know what do you think

```
import java.time.Duration;
import java.time.Period;
```

[1] https://github.com/mockito/mockito/pull/2883/files#diff-98c705b0ceb69f07f9ef4b9e9167ecf84b3617a1e950756298285e56b35df4a8R11-R12

Reply:
Yeah we have talked about this before. Since Mockito 5 now supports 11+, I think this is fine.